e2e/helpers/onboarding.js

```diff
@@ -1,3 +1,32 @@
+// import EnableAuthoritySubscription from '../pages/EnableAuthoritySubscription.po.js';
```

Comment:
dead code, remove, add a `// TODO(#1234): Something something` if there is an outstanding test, raise an issue.